docs/02-app/01-building-your-application/08-testing/03-playwright.mdx

```diff
@@ -121,7 +121,75 @@ Run `npm run build` and `npm run start`, then run `npx playwright test` in anoth
 
 > **Good to know**: Alternatively, you can use the [`webServer`](https://playwright.dev/docs/test-webserver/) feature to let Playwright start the development server and wait until it's fully available.
 
-### Running Playwright on Continuous Integration (CI)
+## Creating your first Playwright component test
+
+Component tests build and mount a specific component without having to bundle your whole application or start a server.
+
+To install Playwright component testing, run the following command:
+
+```bash filename="Terminal"
+npm init playwright@latest -- --ct
+# or
+yarn create playwright --ct
+# or
+pnpm create playwright --ct
+```
+
+Select **React** as your front-end framework. A `playwright` folder will be created in your project, and a `playwright-ct.config.ts` file to enable component testing.
+
+Assuming the same components from the previous section, add a test to validate a component is rendering the expected output:
+
+<AppOnly>
+
+```tsx filename="app/page.test.tsx"
+import { test, expect } from '@playwright/experimental-ct-react'
+import HomePage from './page'
+
+test('display expected content', async ({ mount }) => {
+  // Mount the home page React component
+  const component = await mount(<HomePage />)
+
+  // The home page should have an h1 with "Home"
+  await expect(component.getByRole('heading', { level: 1 })).toHaveText('Home')
+
+  // Validate that a link with the expected URL is present
+  // Following the link is better suited to an E2E test
+  await expect(component.getByRole('link', { name: 'About' })).toHaveAttribute('href', '/about')
+})
+```
+
+</AppOnly>
+
+<PagesOnly>
+
+```jsx filename="pages/index.test.tsx"
+import HomePage from './index'
+
+test('display expected content', async ({ mount }) => {
+  // Mount the home page React component
+  const component = await mount(<HomePage />)
+
+  // The home page should have an h1 with "Home"
+  await expect(component.getByRole('heading', { level: 1 })).toHaveText('Home')
+
+  // Validate that a link with the expected URL is present
+  // *Following* the link is better suited to an E2E test
+  await expect(component.getByRole('link', { name: 'About' })).toHaveAttribute('href', '/about')
+})
+```
+
+</PagesOnly>
+
+> **Warning:**
+> - Playwright component testing is experimental.
```

Comment:
We might want to hold here until this is stable.

Reply:
Oh, i took @samcx's [comment](https://github.com/vercel/next.js/discussions/69329#discussioncomment-10455875) as confirmation to proceed. 

Getting more engagement/feedback on Playwright component testing + Next.js would help a lot in working towards a stable release tho.

Reply:
@sand4rt Didn't realize that Playwright component testing was experimental still! For now, let's put this <picture data-single-emoji=":pr:" title=":pr:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/pr/876af7e7d65e1595.png" alt=":pr:" align="absmiddle"></picture> on pause.

> Getting more engagement/feedback on Playwright component testing + Next.js would help a lot in working towards a stable release tho.

Although this would probably be true, it is safer to not add an experimental feature from a 3rd party to our documentation. Not all of even our own experimental features gets added to our documentation.